strings.txt

```diff
@@ -3865,7 +3864,7 @@ SETUP_GROUP_TITLEFORMATS_DESC
 	FI	Seuraavat kentät ovat käytössäsi sekä yllä, että soittimen asetuksissa. Poistaaksesi kentän, tyhjennä teksi laatikosta, lisätäksesi uuden kentän, syötä teksi tyhjään laatikkoon. Vaihtoehtoja ovat: CT (sisältötyyppi), TITLE, GENRE, TRACKNUM (kappaleen numero kokonaislukuna), FS (tiedoston koko), ARTIST, COMPOSER, CONDUCTOR, BAND, ALBUM, WORK, GROUPING, SUBTITLE, COMMENT, YEAR, SECS (sekunnit), DURATION (minuutit ja sekunnit), VBR_SCALE (vbr/cbr), BITRATE, TAGSIZE, VOLUME (äänenvoimakkuuden säätö), PATH, FILE, EXT (tiedoston pääte), LONGDATE (päiväys, pitkä esitystapa), SHORTDATE (päiväys, lyhyt esitystapa), CURRTIME(aika). Kentät voidaan erottaa toisistaan millä tahansa erottimella. Erottimia käytetään vain mikäli tietokenttiä on valittuna.<br>Tämänhetkistä valintaa tullaan käyttämään WWW-sivuilla.
-	FR	Voici les formats disponibles. Ils peuvent être choisis dans l'interface Web et les platines. Pour supprimer un format, effacez le contenu de la zone de texte. Placez votre curseur dans la zone de texte vide pour saisir un nouveau format. Les codes suivants peuvent être utilisés pour représenter des données : CT (type de contenu), TITLE (titre), GENRE, TRACKNUM (numéro de piste), FS (taille du fichier), ARTIST (artiste), COMPOSER (compositeur), CONDUCTOR (chef d'orchestre), BAND (groupe/orchestre), ALBUM, WORK (œuvre), GROUPING (regroupement / interprétation d'une œuvre), SUBTITLE (titre secondaire), COMMENT (commentaire), YEAR (année), SECS (durée totale en secondes), DURATION (durée en minutes et secondes), BITRATE (débit binaire), LONGDATE (date en cours au format long), SHORTDATE (date en cours au format court), CURRTIME (heure en cours). Vous pouvez utiliser des séparateurs entre chaque élément. Les séparateurs ne sont utilisés que si des données sont présentes.<br>La sélection en cours est le format qui sera utilisé dans les pages Web.
+	FR	Voici les formats disponibles. Ils peuvent être choisis dans l'interface Web et les platines. Pour supprimer un format, effacez le contenu de la zone de texte. Placez votre curseur dans la zone de texte vide pour saisir un nouveau format. Les codes suivants peuvent être utilisés pour représenter des données : CT (type de contenu), TITLE (titre), GENRE, TRACKNUM (numéro de piste), FS (taille du fichier), ARTIST (artiste), COMPOSER (compositeur), CONDUCTOR (chef d'orchestre), BAND (groupe), ALBUM, WORK (œuvre), GROUPING (regroupement / interprétation d'une œuvre), SUBTITLE (titre secondaire), COMMENT (commentaire), YEAR (année), SECS (durée totale en secondes), DURATION (durée en minutes et secondes), BITRATE (débit binaire), LONGDATE (date en cours au format long), SHORTDATE (date en cours au format court), CURRTIME (heure en cours). Vous pouvez utiliser des séparateurs entre chaque élément. Les séparateurs ne sont utilisés que si des données sont présentes.<br>La sélection en cours est le format qui sera utilisé dans les pages Web.
```

Comment:
A French line changed?

There's a free Git client called Sourcetree which would give you a nice GUI to revert such unwanted changes.